docker/Dockerfile

```diff
@@ -1,66 +1,28 @@
-ARG build=local
+FROM node:18-alpine
+ENV name="airseeker-v2"
+LABEL application="airseeker-v2" description="Airseeker V2 container"
+WORKDIR /app
 
-FROM node:18.14.0-alpine3.17 AS environment
-
-ENV name="airseekerv2" \
-    appDir="/app" \
-    buildDir="/build"   
-ENV packageDir="${buildDir}/package"
-
-# Build preparation
-FROM environment AS preparation
-
-WORKDIR ${buildDir}
-
-RUN apk add --update --no-cache git
-
-# Install pnpm
+# Install pnpm globally (needs root privilidges).
 RUN npm install -g pnpm
 
-# Source preparation - local
-FROM preparation as sourcelocal
-
-COPY . ${buildDir}
-
-# Source preparation - git
-FROM preparation as sourcegit
-
-ARG branch=main
-ARG repository=https://github.com/api3dao/airseeker-v2.git
-
-RUN git clone --single-branch --branch ${branch} ${repository} ${buildDir}
-
-# Production dependencies
-FROM source${build} AS deps
-
-RUN pnpm install --prod --no-optional --ignore-scripts
-
-# Production source code
-FROM source${build} AS build
-
-RUN pnpm install && \
-    pnpm build && \
-    pnpm pack && \
-    mkdir -p ${packageDir} && \
-    tar -xf *.tgz -C ${packageDir} --strip-components 1
-
-# Result image
-FROM environment
-
-WORKDIR ${appDir}
-
-LABEL application=${name} \
-    description="AirseekerV2 container"
-
-
-COPY --from=deps ${buildDir}/node_modules ./node_modules
-COPY --from=build ${packageDir} .
-
-# Create Airseeker user
-RUN addgroup -S ${name} && \
-    adduser -h ${appDir} -s /bin/false -S -D -H -G ${name} ${name} && \
-    chown -R ${name} ${appDir}
-
-USER ${name}
-
-CMD ["node", "dist/index.js"]
\ No newline at end of file
+# Create a new user and group to run the application with reduced priviledges.
+RUN addgroup -S airseeker && \
+    adduser -h /app -s /bin/false -S -D -H -G airseeker airseeker && \
```

Comment:
It's probably unnecessary to use a separate user for this in the container? I think most services use root inside docker containers or pass-through the user and group ids from the launch shell. The root-in-container process's root status shouldn't translate into root on the host machine. Feel free to educate me 😆

Reply:
It one of the best practices. You can read a bit more in:
- https://stackoverflow.com/a/68190847
- https://snyk.io/blog/10-best-practices-to-containerize-nodejs-web-applications-with-docker/
- (and many more)

Running the service as a non root user is quite simple to do so it's better to stick with the best practice.

Reply:
Thanks for this 👌